.gitignore

```diff
@@ -2,3 +2,5 @@ target
 **/*.rs.bk
 Cargo.lock
 orig.*
+/.idea
```

Comment:
I think `.gitignore` niceties may be out of scope for this PR, although I would appreciate them myself and I think a separate PR would be welcome.

Thank you for keeping the routing example in line with the rest of the project.